Booking.cs

```diff
@@ -0,0 +1,206 @@
+﻿using static AirportTicketBooking.Flight;
+namespace AirportTicketBooking
+{
+    internal class Booking
+    {
+        public enum BookedStatus
+        {
+            BookedSuccefully = 1,
+            BookedFaild = 0,
+        }
+        public List<Flight> flights;
+        private List<BookingDetails> bookings;
+
+        public Booking(string filepath)
+        {
+            flights = FlightFileImporter.ImportFlightsFromCsv(filepath);
+            bookings = new List<BookingDetails>(); 
+        }
+
+        public BookedStatus BookFlight(Passenger passenger, Flight flight)
+        {
+            if (flight.NumberOfSeats > 0)
+            {
+                if (passenger.BookedFlights.Contains(flight))
+                {
+                    Console.WriteLine("You have already booked this flight.");
+                    return BookedStatus.BookedFaild;
+                }
+                passenger.BookedFlights.Add(flight);
+                BookingDetails bookingDetails = new BookingDetails
+                {
+                    FlightNum = flight.FlightNum,
+                    NumberOfSeats =flight.NumberOfSeats,
+                    DepartureAirport=flight.DepartureAirport,
+                    ArrivalAirport=flight.ArrivalAirport ,
```

Comment:
update all equals to have a space before and a space after
```suggestion
                    ArrivalAirport = flight.ArrivalAirport ,
```